phq/kafka/payload.py

```diff
@@ -76,12 +74,12 @@ def unpack_kafka_payload(message):
     return item, ref
 
 
-def pack_kafka_payload(svc, item, refs=[]):
+def pack_kafka_payload(svc, item, refs):
```

Comment:
good catch ! I haven't saw this, which mean we potentially have issue with our current `pack_kafka_payload`, and probably weird or strange ref in the pipeline.